lib/lists.nix

```
@@ -1864,6 +1868,44 @@ rec {
   */
   unique = foldl' (acc: e: if elem e acc then acc else acc ++ [ e ]) [ ];
 
+  /**
+    Removes duplicate strings from the `list`. O(n log n) complexity.
+
+    :::{.note}
+    Order is not preserved
```

```suggestion
    Order is not preserved.
```

I don't think the first line is special?